app/routers/librarian.py

```diff
@@ -0,0 +1,143 @@
+from fastapi import APIRouter, HTTPException, Depends
+from sqlalchemy.orm import Session
+from app.models import Book as SQLAlchemyBook, Member as SQLAlchemyMember
+from app.schemas import Book as PydanticBook, Member as PydanticMember
+from app.models import Librarian as SQLAlchemyLibrarian
+from app.schemas import (
+    Librarian as PydanticLibrarian,
+    LibrarianCreate as PydanticLibrarianCreate,
+)
+from app.database import get_db
+
+router = APIRouter()
+
+
+@router.post("/add")
+async def add_librarian(
+    librarian: PydanticLibrarianCreate, db: Session = Depends(get_db)
+):
+    db_librarian = SQLAlchemyLibrarian(**librarian.dict())
+    db.add(db_librarian)
+    db.commit()
+    db.refresh(db_librarian)
```

Comment:
Same again, please move all db operations to crud module. If you need to create functions for them you can. Be careful to code repetititon.

Reply:
You did right in the library router. Let's stick to a standard.